pkg/workloads/k8s.go

```diff
@@ -150,7 +165,8 @@ func UserPodContainers(api spec.API) ([]kcore.Container, []kcore.Volume) {
 			containerResourceLimitsList["nvidia.com/gpu"] = *kresource.NewQuantity(container.Compute.GPU, kresource.DecimalSI)
 		}
 
-		containerVolumeMounts := append(defaultVolumeMounts(requiresKubexit), containerMounts...)
+		containerVolumeMounts := containerMounts
+
 		if container.Compute.Inf > 0 {
```

Comment:
Neuron / RTD can be removed. Inferentia will be supported exactly like we support GPUs, with two changes:

* we will automatically request the appropriate amount of hugepages (requests and limits)
* we will automatically add the security capabilities